packages/react-native/React/Views/ScrollView/RCTScrollView.h

```diff
@@ -48,6 +48,7 @@
 @property (nonatomic, assign) BOOL snapToEnd;
 @property (nonatomic, copy) NSString *snapToAlignment;
 @property (nonatomic, assign) BOOL inverted;
+@property (nonatomic, assign) CGRect firstResponderFocus;
```

Comment:
We should document what this rect should be relative to, right now it seems to be to be the window, but should it not be the scrollview itself?

Can you test with scrollviews that do not to take up the entire screen size?